src/components/GameCards/GameCardDefault.vue

```diff
@@ -16,12 +16,22 @@
 
       <i class="fas fa-grip-vertical game-drag-handle" />
 
-      <progress
-        v-if="gameProgress"
-        max="100"
-        :value="gameProgress"
-        @click="openDetails"
-      />
+      <div class="game-progress">
+        <progress
+          v-if="gameProgress"
+          class="small"
+          max="100"
+          :value="gameProgress"
+          @click="openDetails"
+        />
+
+        <span
+          v-if="gameProgress"
+          class="progress-bar-label"
+        >
+          {{ gameProgress }}%
+        </span>
```

Comment:
what do you think about showing the percentage on top of the progress bar to save space

kind of like this:

<img width="444" alt="Screen Shot 2019-12-18 at 10 46 40 AM" src="https://user-images.githubusercontent.com/645310/71110158-fdc1f280-2183-11ea-9d3e-73a0cae89e60.png">

Reply:
I'm trying to do that right now, except I'm struggling to make it work correctly. The text color needs to be readable for any background color, and also behave correctly with `mix-color-blend`. I'm not getting the desired results yet.

I'll get there eventually.

Reply:
![Screenshot 2019-12-19 at 3 57 46 PM](https://user-images.githubusercontent.com/984069/71183484-5a9ed500-2278-11ea-8e72-d14a7f28ba57.png)

Reply:
I'll have to keep searching for a solution for this over the weekend. I don't like that the percentage is barely readable when it's not on the yellow background.

Maybe something with the percentage being rendered twice but with different colors and some voodoo sprinkled on top.

Reply:
Sounds good. Feel free to remove percentage and do it in a different PR as well, up to you.